pages/pets/show.tsx

```diff
@@ -2,25 +2,25 @@ import React, { useEffect, useState } from 'react'
 import axios from 'axios';
 import update from "immutability-helper";
 
-interface GamesShowProps {
+interface PetsShowProps {
   id?: string
 }
 
-const GamesShow:React.FC<GamesShowProps> = () => {
-  const [game, setGame] = useState<any>('')
+const PetsShow:React.FC<PetsShowProps> = (props) => {
+  const [pet, setPet] = useState<any>('')
```

Comment:
I think we want to retype this to be a `Pet | undefined`. can't remember exactly how to do that but it'll be something like `useState<Pet>()`. I think this means that since the default is undefined, both a Pet type and an undefined type are ok.

Pet then needs to be imported from `models/pet.ts`